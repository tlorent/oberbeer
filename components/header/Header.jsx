import React from 'react';
import Link from 'next/link';

import '../header/header.scss';

const Header = () => (
  <header className="header">
    <Link href="/">
      <a className="header__logo">
        Oberbeer
      </a>
    </Link>
  </header>
);

export default Header;