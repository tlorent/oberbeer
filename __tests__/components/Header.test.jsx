import React from 'react';
import { shallow } from 'enzyme';

import Header from '../../components/header/Header';

test('should render Header correctly', () => {
  const wrapper = shallow(<Header />);

  expect(wrapper.find('a').text()).toBe('Oberbeer');

  expect(wrapper).toMatchSnapshot();
});
