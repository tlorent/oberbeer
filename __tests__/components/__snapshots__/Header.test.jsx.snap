// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header
  className="header"
>
  <Link
    href="/"
  >
    <a
      className="header__logo"
    >
      Oberbeer
    </a>
  </Link>
</header>
`;
